assets/js/app/Instance.js

```diff
@@ -18,46 +18,57 @@ LocusZoom.Instance = function(id, datasource, layout, state) {
     this.initialized = false;
 
     this.id = id;
-    this.parent = LocusZoom;
     
     this.svg = null;
 
-    // The _panels property stores child panel instances
-    this._panels = {};
+    // The panels property stores child panel instances
+    this.panels = {};
     this.remap_promises = [];
+
+    // The layout is a serializable object used to describe the composition of the instance
+    this.layout = layout || JSON.parse(JSON.stringify(LocusZoom.DefaultLayout));
```

Comment:
Well, there's the idea what we want to be able to look at different phenotype at the same time somewhat soon. We've talked about doing this by adding panels or data layers on the fly so I think we will need to be ready for this soon (but not immediately). So if we want to add panels, do we do so ny editing a layout object, or do we do so by calling some sort of addPanel. I agree that this doesn't have to happen on this PR before merge, it's just something that isn't just hypothetical.